src/index.css

```
@@ -1,17 +1,27 @@
 @import url(fonts.css);
 @import url('https://fonts.googleapis.com/css2?family=JetBrains+Mono:ital,wght@0,100..800;1,100..800&display=swap');
-@import url(https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined&icon_names=dark_mode);
+@import url('https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined&icon_names=dark_mode');
```

Stylelint (17.2.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined&icon_names=dark_mode')" to be "'https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined&icon_names=dark_mode'" (import-notation)

(import-notation)